src/dragonball/src/dbs_virtio_devices/src/lib.rs

```diff
@@ -155,6 +181,9 @@ pub enum Error {
     /// Guest gave us a descriptor that was too big to use.
     #[error("descriptor length too big.")]
     DescriptorLengthTooBig,
+    /// Error from the epoll event manager
+    #[error("dbs_utils error: {0}.")]
```

Comment:
```suggestion
    #[error("dbs_utils error: {0:?}.")]
```

Use it to print error stack.

Reply:
Fixed.